### ISSUE TITLE
normalise.py - add batch option

### ISSUE THREAD

**Issue by raecasey** [kieranjol/IFIscripts#321]
to be able to batch normalise

when a particular set of circumstances are met:
- no supplements
- individual standalone files, not in sips or aips or any other folder structures, but listed in a parent folder
- not yet oe'd but a list of sequential oe numbers reserved and available to be assigned

first oe number in sequence must be entered

**Comment by kieranjol** [comment]
Sounds good to me!

**Comment by kieranjol** [comment]
Ok so this is a bit more difficult that expected as the current behaviour (which is not intentional), is that 
`normalise.py -sip -i path/to/folder -o path/to/output`
will create one OE folder, and then one UUID package within that OE folder per source object.

So the way forward is to revert this behaviour and add proper batch functionality.

**Comment by kieranjol** [comment]
However, if time is critical, the easiest thing for me to do is to make a temporary batchnormalise script.
This would be a quick, dirty fix that would just launch individual normalise.py processes and it would auto-increment the OE number.
@raecasey Let me know how you're fixed time-wise and I can do one or the other.

**Comment by kieranjol** [comment]
OK so it turns out that the better fix was easier than expected. Can you test? More info in the pull request. https://github.com/kieranjol/IFIscripts/pull/322

**Comment by raecasey** [comment]
hi
yes, - can test today and revert
thanks!

**Comment by raecasey** [comment]
ok, - so i'm running a test now and it looks like it works well.
the command input is exactly the same as if i'm just putting in a single file except i'm putting in a parent directory that contains multiple single files. 

it prompted me for an oe number and just entered the first oe number i had reserved out of 4.

it's running through them incrementally now. The script didn't tell me anything so it's a bit of a leap of faith, but i can see the incremental oe numbers filling up in the destination directory.

it's a live test, - 2 out of the 4 multiple files are processed and they processed successfully. I'll let you know when the other two are finished if it was completely successful.

thanks for this, - it works really well. 
my only concern is the instruction to batch normalise might be confusing as it is the same command as a single input. there might be room for human error.
also, - potential for human error again, - the person running the script would have to be very sure of what they are doing in terms of reserving the correct amount of oe numbers and checking in to make sure the individual packages are being created in the destination folder as it does feel  a little like a leap of faith. - 
maybe, if possible, a list of the numbers that are to be assigned  and a y/n answer might be helpful?

R

**Comment by kieranjol** [comment]
Good point.i think this is doable.
For the sake of merging this into master,how would you feel about merging
it if we are both happy with the state of the packages that this pull
request creates?
Then I could add the user prompts later.if you'd prefer to wait until both
are done, that's cool.

**Comment by raecasey** [comment]
hey

yes, - i'm happy with merging.
the 4 packages were successfully normalised.
i can tip away using this - it will be very helpful to me, - thank you.
i'd like us to get the prompts and helper set in place before gavin starts using it. I'll let you know in advance if i think he's going to need to start batch normalising.
cheers
R

**Comment by kieranjol** [comment]
Awesome,I'm happy to merge as well.
